### ISSUE TITLE
java.lang.RuntimeException: Unable to create application org.electroncash.electroncash3.App: kotlin.KotlinNullPointerException

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#1563]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6452)
	at android.app.ActivityThread.access$1300(ActivityThread.java:219)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1859)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7343)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:933)
Caused by: kotlin.KotlinNullPointerException
	at org.electroncash.electroncash3.SettingsKt$initSettings$2.onChanged(Settings.kt:30)
	at org.electroncash.electroncash3.SettingsKt$initSettings$2.onChanged(Unknown Source:2)
	at android.arch.lifecycle.LiveData.considerNotify(LiveData.java:109)
	at android.arch.lifecycle.LiveData.dispatchingValue(LiveData.java:121)
	at android.arch.lifecycle.LiveData.access$400(LiveData.java:59)
	at android.arch.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:416)
	at android.arch.lifecycle.LiveData.observeForever(LiveData.java:207)
	at org.electroncash.electroncash3.SettingsKt.initSettings(Settings.kt:28)
	at org.electroncash.electroncash3.App.onCreate(App.kt:47)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1182)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6447)
	... 8 more
java.lang.RuntimeException: Unable to create application org.electroncash.electroncash3.App: kotlin.KotlinNullPointerException
```

Reporter
------------

This issue was reported by 1 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 4.0.8-0  | 3.6.5 (default, Dec  7 2018, 14:06:05)  [GCC 4.9] | Android 10 on Google Android SDK built for x86 (generic_x86) | unknown | en_US | 

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by cculianu** [comment]
@mhsmith Ping

**Comment by mhsmith** [comment]
I never released a 4.0.8, only 4.0.7 pre-release builds and 4.0.9 currently on Google Play. This appears to be someone making their own build of something in between (possibly with their own modifications) and running it on an emulator.

Unless it happens with the production build, I don't think any action is required.

**Comment by cculianu** [comment]
Ah ok, will close. Thanks for the info.
